integration-tests/test/shared/utils.ts

```diff
@@ -95,8 +95,8 @@ export const fundUser = async (
 ) => {
   const value = BigNumber.from(amount)
   const tx = recipient
-    ? gateway.depositTo(recipient, { value })
-    : gateway.deposit({ value })
+    ? gateway.depositTo(recipient, '0x', 0, { value, gasLimit: 500_000 })
```

Comment:
Interesting, is this broken if gaslimit is left unspecified?

Reply:
I believe so, yes.

Reply:
can retest upon request

Reply:
Actually, I was wrong, it works without. 
I was thought it was required, and at some point I think I was in a state that it was, but I cannot repro now.